### ISSUE TITLE
[Bug] Persistent Notification Badge on Springboard (Home Screen) can't be cleared

### ISSUE THREAD

**Issue by wondergit113** [arichorn/uYouPlusExtra#127]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

- iOS/iPadOS version: iOS 17.0.1
- Device model: iPhone 13 Pro
- Sideload tool (AltStore, Sideloadly, TrollStore,...): AltStore 2.0b4, AltServer 1.7RC
- The specific version of uYouPlus (**latest or newest is NOT a version number!**): 18.35.4 - 3.0.1

### Describe the bug

I have a notification badge for the uYouPlus install of YouTube. I have tried removing the app, restarting the phone, re-installing the app and cannot get the notification badge to clear.

### Steps to reproduce the issue

1. Install uYouPlus
2. Notification Badge appears
3. Unable to clear notification badge

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by wondergit113** [comment]
Yeah I've tried that but the notification badge persists on uYou. Even updated today and tried again.

**Comment by yodaluca23** [comment]
> Yeah I've tried that but the notification badge persists on uYou. Even updated today and tried again.

I get so many YouTube notifications I just have badges turned off for the app. I don't know if that's a viable solution for you, but you could do that.

**Comment by Hex34568** [comment]
> There isn't an ability to turn off notifications for the uYou version of the app that's installed.

The best solution to this would be using another fresh developer account to sideload it or you can simply delete the Youtube app then reinstall it when the appid expires idk why altstore does this probably the appid doesn’t work properly could blame apple for this
